actions/src/test/com/redhat/service/bridge/actions/kafkatopic/KafkaTopicActionValidatorTest.java

```diff
@@ -10,7 +10,7 @@
 import com.redhat.service.bridge.infra.models.dto.BridgeDTO;
 import com.redhat.service.bridge.infra.models.dto.ProcessorDTO;
 import io.quarkus.test.junit.QuarkusTest;
-import org.junit.jupiter.api.Assertions;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Please check formatting

@r00ta 
As far as I remember reformat should also work for import static. Am I wrong?

Reply:
Fixed :+1: